internal/cortex/frontend/transport/handler.go

```diff
@@ -87,11 +121,22 @@ func NewHandler(cfg HandlerConfig, roundTripper http.RoundTripper, log log.Logge
 			Help: "Size of all chunks fetched to execute a query in bytes.",
 		}, []string{"user"})
 
+		h.totalQueries = promauto.With(reg).NewCounter(prometheus.CounterOpts{
+			Name: "cortex_queries_total",
+			Help: "Total number of queries.",
+		})
```

Comment:
Let's remove this one. We have another existing counter for total queries.
<img width="1395" alt="image" src="https://github.com/databricks/thanos/assets/129140017/bae409a9-8a36-4680-8e4d-4819293085c8">